RELEASE.md

```diff
@@ -21,17 +21,37 @@ Releases can be triggered in two ways:
+3. The statics assets will be published and added to a tag that has been released via the [release static assets action](.github/workflows/docker_release.yml)
+4. We are currently using [https://github.com/release-drafter/release-drafter](https://github.com/release-drafter/release-drafter) to generate release notes. This is a github action that generates draft releases note that can be added to the released tag. Please add the labels specified within [release-drafter](.github/release-drafter.yml) to categorize different pull requests to ensure that any changes are categorized correctly. You can also use the changelog generated for this content.
+5. Update the draft release notes with any changes required and tie it to the released tag.
+6. For the release notes use the below format. Please review some of the releases to check the format used [https://github.com/americanexpress/one-app/releases](https://github.com/americanexpress/one-app/releases)
+
+  ```
+      [Paste changelog entries here]
+    
+      ### Docker Images
+
+      #### Developer Image
+      [one-app-dev/version-released](https://hub.docker.com/layers/oneamex/one-app-dev/link-to-the-docker-tag)
+
+      #### Production Image
+      [one-app/version-released](https://hub.docker.com/layers/oneamex/one-app/link-to-the-docker-tag)
+
+      ### One App Statics
+      https://www.npmjs.com/package/@americanexpress/one-app-statics
 
-3. We are currently using [https://github.com/release-drafter/release-drafter](https://github.com/release-drafter/release-drafter) to generate release notes. Please add the labels specified within [release-drafter](.github/release-drafter.yml) to categorize different pull requests. Update the draft release notes and tie it to the release tag above, you can also link this to different artifacts. The statics assets will be published and added to a tag that has been released.
+  ```
+7. 
 
 ## Manual release process
 
 This process can be used to make ad hoc releases outside of wednesday release cycle.
 
- 1. Run `npm run release` locally within your branch, this would update the changelog, [package.json](package.json), [package-lock.json](package-lock.json)and [one-app-statics package.json](one-app-statics/package.json) with the new version to be released. Push your changes and create a pull request to master.
- 2. When the changes are merged and reviewed. The same process from step 3 above will be followed.
+ 1. Run `npm run release` locally within your branch, this would update [package.json](package.json), [package-lock.json](package-lock.json)and [one-app-statics package.json](one-app-statics/package.json) with the new version to be released. 
+ 2. Run `npm run release:changelog` to generate and update the changelog.
+ 3. Push your changes and create a pull request to master.
```

Comment:
```suggestion
 3. Commit using the `chore(release): X.X.X` message format and push your changes
 and create a pull request to master.
```

Reply:
Would this need to be on a specific branch?

Reply:
nope, although for consistency using the `prepare-release` branch would be a better suggestions. I'll mention that